src/libraries/icubmod/usbCamera/linux/V4L_camera.cpp

```diff
@@ -160,6 +226,47 @@ bool V4L_camera::open(yarp::os::Searchable& config)
     return true;
 }
 
+
+int V4L_camera::getRgbHeight(){
+    return height();
+}
+
+int V4L_camera::getRgbWidth(){
+    return width();
+}
+
+bool V4L_camera::setRgbResolution(int width, int height){
+    param.width=width;
+    param.height=height;
+    return true;
+}
+
+bool V4L_camera::getRgbFOV(double &horizontalFov, double &verticalFov){
+    horizontalFov=param.horizontalFov;
+    verticalFov=param.verticalFov;
+    return configFx && configFy;
+}
+
+bool V4L_camera::setRgbFOV(double horizontalFov, double verticalFov){
+    param.horizontalFov=horizontalFov;
+    param.verticalFov=verticalFov;
+    return true;
+}
+
+bool V4L_camera::getRgbIntrinsicParam(yarp::os::Property &intrinsic){
+    intrinsic=param.intrinsic;
+    return configIntrins;
+}
+
+bool V4L_camera::getRgbMirroring(bool &mirror){
+    mirror=param.mirror;
+    return true;}
+
+bool V4L_camera::setRgbMirroring(bool mirror){
+    param.mirror=mirror;
+    return true;
```

Comment:
This method shall try to set the HW mirroring option of the camera, if available.